tests/api_test/api_test.cpp

```diff
@@ -1128,7 +1131,9 @@ TEST_CASE("bpf_get_process_start_key_udp", "[helpers]")
 
             // IPv6 test.
             connect_send_udp_ipv6_traffic();
-
+            
+            std::cout << "bpf_map_lookup_elem(process_start_key_map) key: " << key << "\n";
+            REQUIRE(bpf_map_lookup_elem(bpf_map__fd(map), &key, &value) == 0);
```

Comment:
does the test need to cleanup these maps before more tests are run?

Reply:
after the second access no, because we unload the program at the end of the test case which to my understanding deletes and clears the map. We do it after the first access because the test program that calls the ext helper function has a map with a max size of one, so we clear it out first before sending the next round of traffic.